user/details/script.js

```diff
@@ -661,6 +659,46 @@ function showContent() {
   section3.classList.remove('hide');
 }
 
+function ifUserNotSuperUser() {
+  const accordionTabs = document.querySelectorAll('.accordion');
+  const accordionIcon = document.querySelectorAll('.accordion-icon');
+  const toolParent = document.querySelectorAll('.tool-parent');
+  accordionIcon.forEach((icon) => {
+    icon.src = '/user/images/lock-icon.svg';
+    icon.classList.add('accordion-icon');
+    icon.style.width = '60%';
```

Comment:
style property applies an inline style to the selected element, therefore can we please use the existing class to do this?
The drawbacks of inline style here can be if any developers apply the `width` using class or id, they won't be able to do it due to higher specificity of inline styles.

Reply:
ok i got it